core/src/block.rs

```diff
@@ -232,6 +233,12 @@ impl<'x> OpenBlock<'x> {
             e
         })?;
         self.block.header.set_state_root(state_root);
+
+        // Note : Does state always hold validator set?
```

Comment:
Could you change this to "FIXME" like the below description?

> NextValidators do not exist in the static validator set.
This code does not work expected when using the static validator set.
Since we will change Foundry to use only the dynamic validator set, we don't add code that handles the static validator set.
After changing Foundry to use only the dynamic validator set, we should remove this comment.

Reply:
Sounds good but once I tested this, it seems that State always carries validator set as an empty list in case of static mode. (I'm not sure)

Reply:
Technically it doesn't cause a error even now since we use `unwrap_or_default()` in `load_from_state()`. 
So just simple comment will be ok.

> `NextValidators` entry doesn't exit in the state during static-validator-set-mode.
>  It doesn't cause direct error since we use `unwrap_or_default()` here, but should be aware of such problem
>  Remove this comment after we completely omit static-validator-set-mode

Reply:
@majecty updated.